core/src/main/scala/com/salesforce/op/filters/RawFeatureFilterResults.scala

```diff
@@ -88,7 +93,33 @@ case class RawFeatureFilterConfig
   correlationType: CorrelationType = CorrelationType.Pearson,
   jsDivergenceProtectedFeatures: Set[String] = Set.empty,
   protectedFeatures: Set[String] = Set.empty
-)
+) {
+
+  /**
+   * Converts case class constructor to a Map; values converted to String
+   *
+   * @return Map[String, String]
+   */
+  def toStringMap(): Map[String, String] = {
+    (Map[String, String]() /: this.getClass.getDeclaredFields) { (key, value) =>
```

Comment:
oh please, let's not build our own to Map / from Map serializations. we already have a ton of json stuff. just use json4s to serialize the classes. if you need then json converted to map - it's also easily possible.